execution_engine/src/engine_state/mod.rs

```diff
@@ -2192,20 +2214,20 @@ where
     }
 
     /// Gets a trie object for given state root hash.
-    pub fn get_trie_full(&self, trie_key: Digest) -> Result<Option<TrieRaw>, Error>
-    where
-        Error: From<S::Error>,
-    {
-        Ok(self.state.get_trie_full(&trie_key)?)
+    pub fn get_trie_full(&self, trie_key: Digest) -> Result<Option<TrieRaw>, Error> {
+        match self.state.get_trie_full(&trie_key) {
```

Comment:
Nit: How come we can't use the `?` operator here?

Reply:
this function will be deleted in an upcoming PR anyway.